src/umpire/strategy/DynamicPool.cpp

```diff
@@ -42,8 +42,8 @@ DynamicPool::DynamicPool(const std::string& name,
   m_actual_bytes{initial_alloc_bytes},
```

Comment:
This should also get rounded.

Reply:
@davidbeckingsale, were you going to round this too as suggested by @jdahm ?